src/App.tsx

```diff
@@ -1,156 +1,131 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { FilterMode } from './types/FilterMode';
 import { UserWarning } from './UserWarning';
+import { TodoList } from './components/TodoList';
+import { TodosFilter } from './components/TodosFilter';
+import { ErrorMessage } from './components/ErrorMessage';
+import { NewTodoForm } from './components/NewTodoForm';
 
-const USER_ID = 0;
+const USER_ID = 6826;
 
 export const App: React.FC = () => {
+  const [allTodos, setAllTodos] = useState<Todo[]>([]);
+  const [shouldShowError, setShouldShowError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [
+    currentFilterMode,
+    setCurrentFilterMode,
+  ] = useState<FilterMode>(FilterMode.All);
+
+  const showError = (errorText: string) => {
+    setErrorMessage(errorText);
+    setShouldShowError(true);
+    setTimeout(() => {
+      setShouldShowError(false);
+    }, 3000);
+  };
+
+  const handleErrorMessageClose = () => {
+    setShouldShowError(false);
+  };
+
+  const loadTodosFromServer = async () => {
+    setShouldShowError(false);
+
+    try {
+      const todosFromServer = await getTodos(USER_ID);
+
+      setAllTodos(todosFromServer);
+    } catch (error) {
+      showError('Unable to load todos');
+    }
```

Comment:
```suggestion
      setAllTodos(todosFromServer);
    } catch {
      showError('Unable to load todos');
    }
```
you don't use this error